docs/src/main/asciidoc/se/guides/logging.adoc

```diff
@@ -0,0 +1,239 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2025 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Helidon SE Logging Guide
+:description: Helidon logging guide
+:keywords: helidon, logging, MDC, jul, log4j, logback, slf4j
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/se.adoc[]
+
+This guide describes various aspects of logging support in Helidon.
+
+== Contents
+
+* <<Helidon Logging,Helidon Logging>>
+* <<Customizing Logging with Java Util Logging,Customizing Logging with Java Util Logging>>
+* <<log4j and slf4j support,log4j and slf4j support>>
+* <<Mapped Diagnostic Context (MDC) support,Mapped Diagnostic Context (MDC) support>>
+* <<Additional Information,Additional Information>>
+
+== Helidon Logging
+
+Helidon modules use the Java Platform Logging API (`System.Logger`) for logging. Therefore,
+Helidon applications will use Java Util Logging (JUL) as the default log framework.
```

Comment:
I don't know if it matters but Java calls the `java.logging` module, which houses the `java.util.logging` package, the Java Logging API.